src/components/PostsList.tsx

```diff
@@ -1,86 +1,62 @@
 import React from 'react';
-
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
-
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <th> </th>
-        </tr>
-      </thead>
-
-      <tbody>
-        <tr data-cy="Post">
-          <td data-cy="PostId">17</td>
-
-          <td data-cy="PostTitle">
-            fugit voluptas sed molestias voluptatem provident
-          </td>
-
-          <td className="has-text-right is-vcentered">
-            <button
-              type="button"
-              data-cy="PostButton"
-              className="button is-link is-light"
-            >
-              Open
-            </button>
-          </td>
-        </tr>
-
-        <tr data-cy="Post">
-          <td data-cy="PostId">18</td>
-
-          <td data-cy="PostTitle">
-            voluptate et itaque vero tempora molestiae
-          </td>
-
-          <td className="has-text-right is-vcentered">
-            <button
-              type="button"
-              data-cy="PostButton"
-              className="button is-link"
-            >
-              Close
-            </button>
-          </td>
-        </tr>
-
-        <tr data-cy="Post">
-          <td data-cy="PostId">19</td>
-          <td data-cy="PostTitle">adipisci placeat illum aut reiciendis qui</td>
-
-          <td className="has-text-right is-vcentered">
-            <button
-              type="button"
-              data-cy="PostButton"
-              className="button is-link is-light"
-            >
-              Open
-            </button>
-          </td>
-        </tr>
-
-        <tr data-cy="Post">
-          <td data-cy="PostId">20</td>
-          <td data-cy="PostTitle">doloribus ad provident suscipit at</td>
-
-          <td className="has-text-right is-vcentered">
-            <button
-              type="button"
-              data-cy="PostButton"
-              className="button is-link is-light"
-            >
-              Open
-            </button>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </div>
-);
+import classNames from 'classnames';
+import { Post } from '../types/Post';
+
+type Props = {
+  userPosts: Post[];
+  selectedPost: Post | null;
+  setSelectedPost: (value: Post | null) => void;
+};
+
+export const PostsList: React.FC<Props> = ({
+  userPosts,
+  selectedPost,
+  setSelectedPost,
+}) => {
+  function handleSelectPost(currPost: Post) {
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    selectedPost?.id === currPost.id
+      ? setSelectedPost(null)
+      : setSelectedPost(currPost);
+  }
+
+  return (
+    <div data-cy="PostsList">
+      <p className="title">Posts:</p>
```

Comment:
Consider using a more descriptive heading level for accessibility. Ensure that the heading level fits the overall document structure.